src/elm/Data/Multipliers/Limited.elm

```diff
@@ -58,3 +60,12 @@ toResourceLevelMultiplier level multiplierType =
                 Increasable.buildMultiplier 7
             else
                 Increasable.buildMultiplier 1
+
+        DecreaseGlobalCost ->
+            Increasable.buildMultiplier 0.9
+
+        DecreaseLevelCost l ->
+            if l == level then
+                Increasable.buildMultiplier 0.77
+            else
+                Increasable.buildMultiplier 0.77
```

Comment:
Should this be 1 for a no-op multiplier when the level doesn't match?

I wonder if it would be worth extracting a function like `Multiplier.identity` or `Multiplier.noOp` that is a multiplier of 1?

Reply:
Good catch, yes, it should be 1 I think.